pKa_challenge_instructions.md

```diff
@@ -133,13 +140,18 @@ It is important that challenge organizers assign unique microstate IDs and keep
 
 ### Updates on microstates lists
 
-Microstate SMILES strings and microstate IDs can be found in [physical_properties/pKa/microstates/](physical_properties/pKa/microstates/) directory. 
-Due to replicate and missing microstates present in the first release of microstate lists, we have updated `SMXX_microstates.csv` files with necessary corrections in Version 1.4 of this repository. 
-The main correction of this update was the removal of resonance structures that were causing dublicate representation of same microstates. 
+Canonical isomeric SMILES, canonical SMILES, and microstate IDs of microstates can be found in [physical_properties/pKa/microstates/](physical_properties/pKa/microstates/) directory. 
+Due to replicate and missing microstates present in the first release of microstate lists, we have updated `SMXX_microstates.csv` files with necessary corrections in Version 1.5 of this repository. 
+
+The main corrections of the update for Version 1.5  was the removal of resonance structures and geometric isomers that were causing dublicate representation of same microstates. 
```

Comment:
"dublicate" -> "duplicate"

Reply:
"main corrections" -> "main correction". Also, extra space after "1.5".

Reply:
Thank you. I corrected these.